jobs/cjd_sample/src/test/java/SampleTests.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.samples.AutoCompleteSample;
+import com.google.samples.BasicCompanySample;
+import com.google.samples.BasicJobSample;
+import com.google.samples.BatchOperationSample;
+import com.google.samples.CommuteSearchSample;
+import com.google.samples.CustomAttributeSample;
+import com.google.samples.CustomFieldSample;
+import com.google.samples.EmailAlertSearchSample;
+import com.google.samples.FeaturedJobsSearchSample;
+import com.google.samples.GeneralSearchSample;
+import com.google.samples.HistogramSample;
+import com.google.samples.LocationSearchSample;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class SampleTests {
+
+  private ByteArrayOutputStream bout;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @Test
```

Comment:
Last thing - could you please separate the tests of each separate class into a single `@Test` method here? They don't have to be individual files, but these tests are meant to catch changes to the behavior of APIs and it is more useful for them to be broken out based on the API calls.

Thanks,
DZ

Reply:
Done. Might need to run 'kokoro' tests again.